main.js

```diff
@@ -0,0 +1,44 @@
+const comments = [
+  {
+    userName: 'Keg Jonah',
+    userImage: './Assets/other-images/IMG-20220504-WA0018.jpg',
+    comments:
+      'Best food delivery app i have used in nigeria. Absolutly recommended and ill use it over and over agai!',
+  },
+  {
+    userName: 'John Kingsley',
+    userImage: './Assets/other-images/IMG-20220504-WA0018.jpg',
+    comments:
+      'Absolutely my favorite restaurant, everything came in ease and the food tastes really great.',
+  },
+  {
+    userName: 'James Hartley',
+    userImage: './Assets/other-images/IMG-20220504-WA0018.jpg',
+    comments:
+      'I use chopLife frequently and I can say they never dissapoint. The delivery process is very smooth and transparent',
+  },
+  {
+    userName: 'Esosa Constance',
+    userImage: './Assets/other-images/IMG-20220504-WA0018.jpg',
+    comments:
+      'I always use chopLife for my family dinner, My family always asl for it. They are the best!',
+  },
+];
```

Comment:
- [ ] Kindly add two more comments to your array, from the requirements you should have 6 comments, you currently have 4, and they should be in 2 columns just like in the Behance design 👍🏿 

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_281](https://user-images.githubusercontent.com/69638013/175185437-29ad50d0-6e6a-4a78-afd1-a82672d43f27.png)  |  ![Screenshot_280](https://user-images.githubusercontent.com/69638013/175185449-c62637bf-34c9-4852-9c70-fd7fcc7f9ec8.png)